cross/borgbackup/Makefile

```diff
@@ -0,0 +1,17 @@
+PKG_NAME = borgbackup
+PKG_VERS = 1.1.2
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://files.pythonhosted.org/packages/source/b/$(PKG_NAME)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = cross/acl cross/lz4 cross/openssl
+
+HOMEPAGE = https://borgbackup.readthedocs.io/
+COMMENT  = Deduplicating backup program with compression and authenticated encryption
+LICENSE  = Borg
```

Comment:
isn't Borg BSD?

Reply:
I do not know: licence is this https://github.com/borgbackup/borg/blob/master/LICENSE
What shall I put in the Makefile?

Reply:
hm.. seems like on here it's mentioned to be BSD 3 Clause.
https://github.com/borgbackup/borg#main-features (Free and Open Source Software)